.github/workflows/tests.yml

```diff
@@ -22,7 +22,16 @@ jobs:
         #run: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
     steps:
     - uses: actions/checkout@v4
-    - name: setup python
+    - name: setup python 3.12
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.12
```

Comment:
uv is supposed to handle this. is there a problem with the uv python?

Reply:
fixed